CMake/GenerateBindings.cmake

```diff
@@ -99,9 +99,9 @@ set(BINDING_OUTPUTS
 )
 
 # Make the file paths absolute and pointing to the bindings folder:
-set(BINDINGS_FOLDER "${PROJECT_SOURCE_DIR}/src/Bindings/")
+set(BINDINGS_FOLDER "${CMAKE_SOURCE_DIR}/src/Bindings/")
 list(TRANSFORM BINDING_OUTPUTS PREPEND ${BINDINGS_FOLDER})
-list(TRANSFORM BINDING_DEPENDENCIES PREPEND "${PROJECT_SOURCE_DIR}/src/")
+list(TRANSFORM BINDING_DEPENDENCIES PREPEND "${CMAKE_SOURCE_DIR}/src/")
```

Comment:
What happened here?

Reply:
I don't see anything untoward? Sorry if I'm missing something

Reply:
Why was it changed from `PROJECT_SOURCE_DIR` to `CMAKE_SOURCE_DIR`. This would disallow including cuberite as a subfolder.

Reply:
exotic
ok I'll change it back

Reply:
dun dun